### ISSUE TITLE
Create own dictionary

### ISSUE THREAD

**Issue by Mannshoch** [languagetool-org/languagetool#839]
I would like to write some fictional text's with self invented words and also I need a directory where I could add names and Family names. 
Described in [http://wiki.languagetool.org/hunspell-support](http://wiki.languagetool.org/hunspell-support)  I have to modify `\org\languagetool\resource\xx\hunspell\spelling.txt`

I'm looking for a separate saved and empty file to be sure that an Upgrade of Languagetool don't remove my entries and I also be able to get updates of these `spelling.txt` 

as far I understood the wiki-page I have to compile every time I add a word to `spelling.txt` with.
`java -cp languagetool.jar org.languagetool.tools.SpellDictionaryBuilder de-DE /path/to/dictionary.txt org/languagetool/resource/xx/hunspell/xx_xx.info - -o /tmp/output.dict`

Because in this code you name the textfile `dictionary.txt` I'm not sure if I have somehow to setup that my own `spelling.txt` is `spelling.txt: words that the spell checker will ignore and use to generate corrections if someone types a similar word` and also I'm not sure what the mixing of US_EN and de_DE in the example means for the spell-checker. Could be interesting If I wish to  use my spelling.txt in German and English

Could someone please clarify this part in Wiki please?

**Comment by danielnaber** [comment]
> as far I understood the wiki-page I have to compile every time I add a word to spelling.txt

No, you only need to restart LT after you've modified that file.

**Comment by Mannshoch** [comment]
Could you please describe this a bit more detailed?
In which situation do I need to compile and in which situation I don't. 
And also how do I use, if possible, my own spelling.txt?

**Comment by danielnaber** [comment]
You never need to compile your own dictionary. Using your own spelling.txt is not supported yet.
